### ISSUE TITLE
Empty value in select returns index instead

### ISSUE THREAD

**Issue by alesmenzel** [terkelg/prompts#205]
### Describe the bug
Incorrect result value if value in select is an empty string. It returns index instead of the value.

### To Reproduce

```js
// "prompts": "^2.2.1"
const prompts = require('prompts')

async function main () {
	const res = await prompts({
		type: 'select',
		name: 'val',
		message: 'Select:',
		initial: 0,
		choices: [
			{ title: 'something', value: 'some-value' },
			{ title: 'nothing', value: '' },
		],
	})
	console.log(res)
        // returns 1 for nothing, but returns 'some-value' for 'something'
}

main()
```

### Expected behavior
I expect to get the value that I have defined - the empty string.

### System

- OS: [e.g. MacOS 10.14.3] MacOS
- Terminal: [e.g. iTerm, PowerShell] iTerm
- Node version: `node -v` v10.16.0

**Comment by DanielGibbsNZ** [comment]
I believe this was fixed in #188 and released in version 2.3.0.

**Comment by sinedied** [comment]
Could this issue be reopened? I have the exact same issue with v2.4.2
